andela/hc-beef-sauce
# #158532841 Create priority list for user notification

#### What does this PR do?
Allows the owner of a check to pick the priority of each member of the team. This priority will be used to order the alerting of users when a check goes down.

#### Description of Task to be completed?
- The owner should be able to set each team members priority
- Notifications should be sent out to high priority team members first
- After some time notifications should be sent to all team members
- If nag is enabled, the failure of a check should alert all team members

#### How should this be manually tested?
- Create a team 
- Add 2 members and give them different priorities
- Assign them both to one check
- Ping the check and wait for it to go down
- The high priority user should get the first down notification
- The low priority user will get the notification if the high priority user does not resolve it in a specified time period

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/158532841
